src/dashboard/src/components/api/views.py

```diff
@@ -541,6 +522,8 @@ def fetch_levels_of_description_from_atom(request):
     else:
         return get_levels_of_description(request)
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET', 'POST'])
 def path_metadata(request):
```

Comment:
... which would require changes to dashboard/src/media/js/ingest/directory-metadata-form-dialog.js